src/dataServices/dynamoDbDataService.ts

```diff
@@ -158,30 +158,47 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
         };
     }
 
+    // async deleteResource(request: DeleteResourceRequest) {
```

Comment:
I should probably delete the commented code. or?

Reply:
I would. If we ever need to rollback, we can revert the changes made in this commit.